src/Http/Controllers/Traits/BreadRelationshipParser.php

```diff
@@ -10,19 +10,26 @@
 {
     protected $relation_field = [];
 
+    /**
+     * Remove Relationship Field
+     *
+     * @param  DataType $dataType
+     * @param  string   $bread_type BREAD type. Default 'browse'
+     */
     protected function removeRelationshipField(DataType $dataType, $bread_type = 'browse')
     {
-        $forget_keys = [];
+        $_keys = [];  // Forget keys
```

Comment:
Why all the field renaming? I'm perfectly content with the verbose names as they're actually meaningful.  There's a reason we don't name everything `a`, `b`, and `c`

Reply:
Some variable names are so long, harder to read. I renamed variables based on their scope, tried to abbreviate, but I guess it was radical in some cases..
Why don't we abbreviate obvious variables?
Tell what to do with this so we can close this. TY!